src/plugins/intel_gpu/tests/test_cases/mvn_gpu_test.cpp

```diff
@@ -842,7 +843,7 @@ struct mvn_random_test_bsv32 : ::testing::TestWithParam<mvn_basic_test_params> {
 
         auto outputs_opt = net_opt.execute();
         auto output_opt = outputs_opt.at("mvn_opt").get_memory();
-
+        std::cout << "mvn_opt impl info: " << net_opt.get_implementation_info("mvn_opt") << std::endl;
```

Comment:
Do we really need this print?

Reply:
It seems unnecessary because it is forced to use the 'mvn_gpu_b_fs_yx_fsv16_imad' kernel.
I will remove the line.